docs/user/tutorial-asgi.rst

```diff
@@ -463,9 +548,38 @@ functionality. The final version of ``images.py`` reads:
 .. literalinclude:: ../../examples/asgilook/asgilook/images.py
     :language: python
 
-Adding a new thumbnails route in ``app.py`` is left as an exercise for the
-reader (if you get stuck, see the final version of ``app.py`` later in this
-tutorial).
+Adding a new thumbnails :meth:`route <falcon.asgi.App.add_route>` in ``app.py``
+is left as an exercise for the reader.
+
+.. tip::
+    Draw inspiration from the thumbnail URI formatting string:
+
+        .. code:: python
+
+            f'/thumbnails/{self.image_id}/{width}x{height}.jpeg'
+
+    The actual URI template for the thumbnails route should look quite similar
+    to the above.
+
+    Remember that we want to use the
+    :class:`uuid <falcon.routing.UUIDConverter>` converter for the ``image_id``
+    field, and image dimensions (``width`` and ``height``) should ideally be
+    converted to :class:`int <falcon.routing.IntConverter>`\s.
+
+(If you get stuck, see the final version of ``app.py`` later in this tutorial.)
+
+.. note::
+    If you tried to access a non-existent route (e.g., if you forgot to add an
+    intended route, or simply misspelled the URI), the framework would
+    automatically render an ``HTTP 404 Not Found`` response by raising an
+    instance of :class:`~falcon.HTTPNotFound` (unless that exception is
+    intercepted by a
+    :meth:`custom error handler <falcon.asgi.App.add_error_handler>`, or masked
+    by a sink).
```

Comment:
I think we could better specify
"or masked by a sink" -> "or the path is masked by a sink"

Reply:
Agreed. Tbh looking at this again, I'm not sure if "masked" sounds right either.

Reply:
shadowed is maybe an alternative. also matched

Reply:
Reworded.

Reply:
Ah, sorry @CaselIT I reworded without looking at your comment. Nevertheless, I happened to pick one of your suggestions :slightly_smiling_face: